systemtest/src/test/java/io/strimzi/kafka/access/AbstractST.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.kafka.access;
+
+import io.skodjob.testframe.annotations.ResourceManager;
+import io.skodjob.testframe.annotations.TestVisualSeparator;
+import io.skodjob.testframe.resources.ClusterRoleBindingType;
+import io.skodjob.testframe.resources.ClusterRoleType;
+import io.skodjob.testframe.resources.CustomResourceDefinitionType;
+import io.skodjob.testframe.resources.DeploymentType;
+import io.skodjob.testframe.resources.KubeResourceManager;
+import io.skodjob.testframe.resources.NamespaceType;
+import io.strimzi.kafka.access.installation.SetupAccessOperator;
+import io.strimzi.kafka.access.resources.KafkaAccessType;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.TestInstance;
+
+@ResourceManager
+@TestVisualSeparator
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+@SuppressWarnings("ClassDataAbstractionCoupling")
+public abstract class AbstractST {
+    protected final KubeResourceManager resourceManager = KubeResourceManager.getInstance();
+    public final String namespace = "main-namespace";
+    private final String kafkaCrdUrl = "https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/main/packaging/install/cluster-operator/040-Crd-kafka.yaml";
```

Comment:
Is this going to cause issues if Strimzi makes breaking changes, e.g. moves to a new api version for the custom resources? Should we pin it to a specific version?

Reply:
Good point, I will pin it to the same version we have for the Strimzi API (from where are all the builders etc.)